files/acl.auth.php

```diff
@@ -0,0 +1,61 @@
+# acl.auth.php
+# <?php exit()?>
+# Don't modify the lines above
+#
+# Access Control Lists
+#
+# Editing this file by hand shouldn't be necessary. Use the ACL
+# Manager interface instead.
+#
+# If your auth backend allows special char like spaces in groups
+# or user names you need to urlencode them (only chars <128, leave
+# UTF-8 multibyte chars as is)
+#
+# none   0
+# read   1
+# edit   2
+# create 4
+# upload 8
+# delete 16
+#
+*       @ALL    0
+*       @user   1
+aks:ak_computer:*       @ALL    0
```

Comment:
None of this belongs into the role. This should be configurable.

Reply:
done

Reply:
The file is still there, with hardcoded values for a specific configuration.
There should be no file needed at all, this should be configurable using yml.